src/Switch/Switch.less

```diff
@@ -0,0 +1,133 @@
+@import (reference) "../less/utilities";
+
+@grey: #71738A;
+@grey-light: #D6D7DE;
+@grey-dark: #474C5E;
+@green: #49D2A2;
+@green-dark: #018657;
```

Comment:
we also have pre-defined colour variables in `css/colors.less` that we've standardised on: https://clever.github.io/components/#/design/colors
instead of these variables, you can use those directly and pick ones that are close to what you need